### PR TITLE
PS-5682 extra_port & extra_max_connections removal not reflected in docs

### PR DESCRIPTION
**Pull request by patrickbirch**
In threadpool.rst, added a paragraph on upgrading to 8.0.14 and higer and for each variable, a version-info property with the version removed information.

### CODE REVIEW

File: doc/source/performance/threadpool.rst [hunk 1]
```diff
@@ -4,20 +4,20 @@
  Thread Pool
 =============
 
-|MySQL| executes statements using one thread per client connection. Once the number of connections increases past a certain point performance will degrade. 
+|MySQL| executes statements using one thread per client connection. Once the number of connections increases past a certain point performance will degrade.
```
Comment (laurynas-biveinis) [inline doc/source/performance/threadpool.rst]: Perhaps line wrap if this is touched anyway?

File: doc/source/performance/threadpool.rst [hunk 2]
```diff
@@ -154,33 +154,50 @@ This variable can be used to define the number of threads that can use the CPU a
...
      :conf: Yes
      :scope: Global
      :dyn: Yes
      :vartype: Numeric
      :default: 1
-     
+
 This variable can be used to specify the maximum allowed number of connections plus one extra ``SUPER`` users connection on the :variable:`extra_port`. This can be used with the :variable:`extra_port` variable to access the server in case no new connections can be established due to all worker threads being busy or being locked when ``pool-of-threads`` feature is enabled.
 
+
```
Comment (laurynas-biveinis) [inline doc/source/performance/threadpool.rst]: Are the extra newlines intentional?

**Review by laurynas-biveinis — CHANGES_REQUESTED** [review]
Looks good in general, only minor comments

File: doc/source/performance/threadpool.rst (hunk 2 above)
Comment (patrickbirch) [reply doc/source/performance/threadpool.rst]: Removed.

File: doc/source/performance/threadpool.rst (hunk 1 above)
Comment (patrickbirch) [reply doc/source/performance/threadpool.rst]: Sure.

**Pull request closed without merging**